cs/common/netstandard/CMakeLists.txt

```diff
@@ -16,11 +16,11 @@ if (vw_DOTNET_USE_MSPROJECT)
   include_external_msproject(VowpalWabbit.Common ${CMAKE_CURRENT_SOURCE_DIR}/vw.common.csproj)
 else()
   add_custom_target(VowpalWabbit.Common ALL
-    COMMAND ${DOTNET_COMMAND} build ${CMAKE_CURRENT_SOURCE_DIR} -o $<TARGET_FILE_DIR:vw.net.native> -v m --nologo --no-dependencies /clp:NoSummary
+    COMMAND ${DOTNET_COMMAND} build ${CMAKE_CURRENT_SOURCE_DIR} -o $<TARGET_FILE_DIR:vw.net.native> --configuration "$<IF:$<CONFIG:Debug>,Debug,Release>" -v m --nologo --no-dependencies /clp:NoSummary
```

Comment:
Do we need to handle RelWithDebInfo and MinSizeRel?

Reply:
I don't think dotnet supports CMake's idea of RelWithDebInfo and MinSizeRel. The way it is right now, both would be handled as Release if you were to use them. I think that's reasonable, as they are both variants of release builds.

We only build the Nuget package with the Release configuration, anyways.